docs/getting-started/install-zowe-sdks.md

```diff
@@ -0,0 +1,87 @@
+# Zowe SDKs installation
+
+Leverage the Zowe Client Software Development Kits (SDKs) to build client applications and scripts that interface with the mainframe.
+
+The SDKs include programmatic APIs, each of which performs a particular mainframe task. For example, one API package provides the ability to upload and download z/OS data sets. You can leverage that package to rapidly build a client application that interacts with data sets.
+
+The following SDKs are available.
+- Zowe Client Node.js SDK
+- Zowe Client Python SDK *technical preview*
+- Zowe Client Kotlin SDK *technical preview*
+- Zowe Client Java SDK *under development*
```

Comment:
I recall I received approval to mark java sdk as GA it is not in TP or UD status. It is pretty well baked. The Java SDK has been added to the intellj squad team for oversite and support me being the main maintainer. As such, I think the  "Zowe Client Java SDK" should be specified under Node.js without status indicator. Check with the team maybe I am wrong about this.

Reply:
Hey @frankgiordano,
I agree with you and I believe we should update the zowe.org to reflect these changes. 😋 
![image](https://github.com/zowe/docs-site/assets/37381190/aa07120c-c9ce-45f6-afcc-7911aeb620fa)

That being said, I think we should keep them in-sync (`under-development`) until zowe.org shows the GA status. 😋

Reply:
The Zowe.org main page has been updated to reflect that Java and Kotlin SDKs are both GA. Modifying this PR to reflect those changes.